Molio/Source/Domain/UseCase/UserUseCase/DefaultUserUseCase.swift

```diff
@@ -0,0 +1,71 @@
+import Foundation
+
+final class DefaultUserUseCase: UserUseCase {
+    private let service: UserService
+    
+    init(
+        service: UserService
+    ) {
+        self.service = service
+    }
+    
+    func createUser(userID: String, userName: String, imageURL: URL?, description: String?) async throws {
+        let newUser = MolioUserDTO(
+            id: userID,
+            name: userName,
+            profileImageURL: imageURL?.absoluteString,
+            description: description
+        )
+        try await service.createUser(newUser)
+    }
+    
+    func fetchUser(userID: String) async throws -> MolioUser {
+        let userDTO = try await service.readUser(userID: userID)
+        
+        let user = MolioUser(
+            id: userDTO.id,
+            name: userDTO.name,
+            profileImageURL: userDTO.profileImageURL != nil ? URL(string: userDTO.profileImageURL!) : nil,
+            description: userDTO.description
+        )
+        
+        return user
+    }
```

Comment:
P3;
삼항 연산자로 처리해줘서 괜찮을 것 같긴 하지만

```suggestion
func fetchUser(userID: String) async throws -> MolioUser {
    let userDTO = try await service.readUser(userID: userID)

    let profileImageURL: URL?
    if let urlString = userDTO.profileImageURL {
        profileImageURL = URL(string: urlString)
    } else {
        profileImageURL = nil
    }

    let user = MolioUser(
        id: userDTO.id,
        name: userDTO.name,
        profileImageURL: profileImageURL,
        description: userDTO.description
    )

    return user
}
```

이거 어떠신가요

Reply:
상관없습니다! 반영하겠습니다!